src/eos-metrics-instrumentation.c

```diff
@@ -893,7 +893,10 @@ record_network_id_impl (const char *image_version,
   guint32 network_id;
 
   /* Network ID is only needed for analysis on Solutions images */
```

Comment:
Nitpick: this comment is now out-of-date. Should be "on select images" or similar to avoid aliasing the code below.

Reply:
Basically all of these are Solutions deals, no? In any case I added a commit rewording the comments and debug messages.

Reply:
The `fnde` images are for Solutions deals but, pedantically, the `impact` images are more Philanthropy (with the work done by Solutions). In theory, if/when Philanthropy has engineers, they'll do the work and manage the images themselves.

And it might be helpful to clarify the distinction between `solutions` (the image product) and Solutions the company.